### PR TITLE
Elemen desktop location fix

### PR DESCRIPTION
**Pull request by happysalada**
###### Motivation for this change

upon running element desktop there is the following error in the terminal
```
Fetching translation json for locale: en
Could not fetch translation json for locale: 'en' Error: Cannot find module './i18n/strings/en.json'
```

I've also run nixpkgs-fmt, hope that's not a problem.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/element/element-desktop.nix [hunk 1]
```diff
@@ -54,6 +60,7 @@ in mkYarnPackage rec {
     cp -r './deps/element-desktop/res/img' "$out/share/element"
     rm "$out/share/element/electron/node_modules"
     cp -r './node_modules' "$out/share/element/electron"
+    cp $out/share/element/electron/lib/i18n/strings/en_EN.json $out/share/element/electron/lib/i18n/strings/en.json
     cp $out/share/element/electron/lib/i18n/strings/en_EN.json $out/share/element/electron/lib/i18n/strings/en-us.json
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/element/element-desktop.nix:64]:
```suggestion
    cp $out/share/element/electron/lib/i18n/strings/en_EN.json $out/share/element/electron/lib/i18n/strings/en-us.json
    ln -s $out/share/element/electron/lib/i18n/strings/en{,-us}.json
```

File: pkgs/applications/networking/instant-messengers/element/element-desktop.nix [hunk 2]
```diff
@@ -55,6 +61,7 @@ in mkYarnPackage rec {
     rm "$out/share/element/electron/node_modules"
     cp -r './node_modules' "$out/share/element/electron"
     cp $out/share/element/electron/lib/i18n/strings/en_EN.json $out/share/element/electron/lib/i18n/strings/en-us.json
+    ln -s $out/share/element/electron/lib/i18n/strings/en{,-us}.json
```
Comment (yu-re-ka) [inline pkgs/applications/networking/instant-messengers/element/element-desktop.nix:64]:
```suggestion
    ln -s $out/share/element/electron/lib/i18n/strings/en{-us,}.json
```

**Review by yu-re-ka — CHANGES_REQUESTED** [review]
Doesn't build, see my suggested change

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 132765` at 1f1de628 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
  </ul>
</details>

**Comment by happysalada** [comment]
I've squashed the commits, I'm testing one more time locally just in case.
Thank you both for the reviews!

**Pull request merged**